assertk/src/jvmMain/kotlin/assertk/assertions/support/support.kt

```diff
@@ -4,6 +4,7 @@ package assertk.assertions.support
 internal actual fun displayPlatformSpecific(value: Any?): String {
     return when (value) {
         is Byte -> "0x%02X".format(value)
+        is Short -> "0x%04X".format(value)
```

Comment:
Shorts in hex is a little weird I think. Most short usage in my experience is for a bitwise protocol, but it still is usually a decimal value.

Reply:
yeah the more I'm thinking about it the more I'm against it, think I'm going to adjust this one to only the show bytes as hex on all platforms part